treeherder/push_health/tests.py

```diff
@@ -146,13 +139,19 @@ def get_current_test_failures(push, option_map, jobs, investigatedTests=None):
                 'suggestedClassification': 'New Failure',
                 'confidence': 0,
                 'tier': job.tier,
+                'totalFailures': 0,
+                'totalJobs': 0,
```

Comment:
I'm a bit surprised that a field called `totalJobs` ends up excluding results other than `success` and `testfailed`. I'm assuming that this means, eg: jobs that had an exception (like malformed payload) wouldn't be counted here). Is there a better name we could use for this?

Maybe I'm misunderstanding that possible states here, though.

Reply:
cancelled, pending, etc.  the main goal is to count total jobs that ran the test- this is an edge case and makes an assumption that if a job ran 3 times that a given test in that job ran 3 times as well.

Reply:
Ah, I see! So this is a count of individual tests running, not tasks. In that case, I think the naming makes sense since cancelled tasks, etc. don't actually run the tests.

Reply:
maybe I misrepresented it:
totalFailures = total times this test case failed
totalJobs = total completed jobs which the test belongs to

Reply:
Right, sorry. That's what I meant.